### ISSUE TITLE
Port for .net core?

### ISSUE THREAD

**Issue by igeligel** [geel9/SteamAuth#53]
Since it is [now RTM](https://www.microsoft.com/net/core#windows), is there a chance that this library is getting updated to support .net core?

**Comment by Shravan2x** [comment]
Still has no ARM support ;-;

**Comment by igeligel** [comment]
Is this a requirement? 
Anyway we could create the same functionality just in this library with [preprocessor directives](http://stackoverflow.com/questions/3436526/detect-target-framework-version-at-compile-time). Some stuff is not featured in .net core for example HttpWebRequest, this is why a port is really hard.

**Comment by Shravan2x** [comment]
Are you sure HttpWebRequest isn't in .NET Core? I just tried a console app and it seemed to be there.

**Comment by igeligel** [comment]
The problem is that there is no synchronous support. So you need to change some stuff in this library i believe. Probably other dependencies are not given too.

**Comment by Shravan2x** [comment]
I spent some time with it and think we should wait longer till more libraries pop up.

**Comment by igeligel** [comment]
So i want to start with this now. Should i create a new branch for it, a seperate folder or work with [conditional compilation](http://stackoverflow.com/questions/38040466/conditional-compilation-symbol-for-net-core-class-library)?

**Comment by Shravan2x** [comment]
A similar request I made to SteamKit a few days back: https://github.com/SteamRE/SteamKit/issues/299#issuecomment-249801734

TL;DR They think we should wait for .NET Standard 2.0 which would make our job much easier.

**Comment by igeligel** [comment]
It seems like netstandard 2.0 is supporting .net framework 4.6.1. This would be an upgrade of the .net framework in this library. Is geel9 ok with this?

**Comment by Shravan2x** [comment]
Probably will be, I mean - it's way too much work without. Even if SteamAuth managed to remain on 4.5, SteamKit and others will move to 4.6.1. So I don't think it makes sense.

**Comment by Marc3842h** [comment]
As .NET Standard 2.0 got released a while ago and SteamKit als updated to it, is there an update on this issue?

**Comment by MakcStudio** [comment]
Also interested.. Will it be ported to net standard 2?
